db/migrate/20250210143408_add_comment_to_controls.rb

```diff
@@ -0,0 +1,17 @@
+class AddCommentToControls < ActiveRecord::Migration[8.0]
+  def up
+    # Fun fact: MySQL doesn't support default values for TEXT columns.
+    #
+    # Given that we're adding a new TEXT column and want it to be non-null, we have to do a bit of a
+    # silly dance here.
```

Comment:
I would add this to commit messages instead, perhaps with a note about which version of MySQL you're grappling with. You never know, it could change in a future release.